### DISCUSSION TITLE
iPad Pro 11 with M4 can't change Cover, Fit, Fill mode

### DISCUSSION THREAD

**Discussion by tthkbw** [blinksh/blink#2010] [Support]
I have used blink on an iPad Pro 12.9 Gen 3 for years and love it. I just upgraded to an ipad Pro 11 with M4. I migrated everything from the old iPad Pro 12.9. I am running iPadOS 17.5 on both machines.

On the iPad Pro 11 M4, the upper left corner of the terminal is cut off by the rounded edge of the display. Most of the character at the upper left corner is missing, as is part of the second character on the first line. 

Seems like I should be able to fix this with config->appearance->Cover, Fit, Fill option. However, when I use config to change those, there is no difference. I have made the change, created new shells with double tap and the problem persists. The same is true if I make a change to the config, then exit the program and restart. Changing these setting via config doesn't appear to do anything. I can make theme changes and see them implemented when I create a new blink shell with two-finger tap.

Tapping with three fingers doesn't work to change Cover, Fit, Fill. I initially had this gesture mapped to zoom the display in iOS, but have disabled that. Then what I got with three finger tap was the system copy paste menu. If I disable Productivity Gestures in Settings->Multitasking and Gestures, then three finger tap does nothing. 

Any suggestions?

**Comment by carloscabanero** [comment]
Hey! Thanks for reaching out. This is covered here: https://github.com/blinksh/blink/issues/2009

Also, just a heads up, we changed the "three finger tap" gesture to "double tap command" or "double tap on home menu": https://docs.blink.sh/#gestures.

This should be out for 17.3.0, right before WWDC.
